### PR TITLE
feat: sharebuttons

### PR DESCRIPTION
**Pull request by koilebeit**
# Pull request

## Proposed changes

Add sharebuttons to blog post pages (#67).
To be discussed:

- where should the sharebuttons be available?
- placement and layout

## Types of changes

- [x] New feature (non-breaking change which adds functionality).
- [ ] Enhancement (non-breaking change which enhances functionality)
- [ ] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[README](./README.md)** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Summary by CodeRabbit

- **New Features**
	- Integrated social media share buttons on blog post pages, including Email, LinkedIn, WhatsApp, Facebook, and a custom share button for easy content sharing.

### CODE REVIEW

File: src/routes/blog/[slug]/+page.svelte [hunk 1]
```diff
@@ -33,4 +44,19 @@
 		</hgroup>
 		<svelte:component this={data.content} />
 	</article>
+	<div class="box-border text-end text-sm">
```
Comment (maehr) [inline src/routes/blog/[slug]/+page.svelte:47]: @koilebeit thank you very much for this PR. I'd order the platforms along their usage in our peer group which IMHO would be
- email
- whatsapp
- facebook
- linkedin
- x
Are there any numbers on telegram?

Comment (koilebeit) [reply src/routes/blog/[slug]/+page.svelte:47]: Thanks for the review. I can change the order of the buttons.

But there is still the question of the exact placement and layout/design of the share buttons. Also the question of whether the buttons should only appear in the blog posts or also in the blog overview or elsewhere on the homepage?

Comment (maehr) [reply src/routes/blog/[slug]/+page.svelte:47]: I'd move the bar up, over the blog content. But we should ask @mtwente for design questions

**Comment by mtwente** [comment]
What are the benefits of using the svelte-share-buttons-component? In [here](https://github.com/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch/blob/701107c56bfcec7a83304c05aa6a0a1d04859334/src/routes/%2Blayout.svelte) we use the svg paths directly for Instagram, Twitter and the menu icons. I think it's easier to style the icons (colours, dark mode) directly this way.

**Comment by maehr** [comment]
> What are the benefits of using the svelte-share-buttons-component? In [here](https://github.com/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch/blob/701107c56bfcec7a83304c05aa6a0a1d04859334/src/routes/%2Blayout.svelte) we use the svg paths directly for Instagram, Twitter and the menu icons. I think it's easier to style the icons (colours, dark mode) directly this way.

It is already implemented. If you want to implement it yourself, I am all ears.

**Review by maehr — APPROVED** [review]

**Pull request merged**